go/arrow/array/numericbuilder.gen.go.tmpl

```diff
@@ -203,6 +203,21 @@ func (b *{{.Name}}Builder) AppendValueFromString(s string) error {
       return err
     }
     b.Append(arrow.Date64FromTime(tm))  
+  {{else if or (eq .Name "Dictionary") -}}
```

Comment:
why don't I see these changes in the `numericbuilder.gen.go` file? do you need to re-run go generate? Also, I don't think Dictionary goes through this code path at all? Or am I wrong?

Reply:
Yep, seems like it. I'll remove the references for dictionary